test/integration/verify/controls/azurerm_key_vaults.rb

```diff
@@ -0,0 +1,10 @@
+resource_group    = attribute('resource_group', default: nil)
+vault_name        = attribute('key_vault_name', default: nil)
+
+control 'azurerm_key_vaults' do
+
+  describe azurerm_key_vaults(resource_group: resource_group) do
+    it                    { should exist }
+    its('names')          { should include vault_name }
+  end
+end
```

Comment:
There's a lot of unnecessary whitespace in these test files. The process is to write each line as normal, but then push out right-hand segments to match the furthest one.

So if you have some assignments like these:
```ruby
red = [255,0,0]
yellow = [0,255,0]
blue = [0,0,255]
```
then the furthest column is in the `yellow` assignment. Use spaces to push out the assignment operators for `red` and `blue` to match.
```ruby
red    = [255,0,0]
yellow = [0,255,0]
blue   = [0,0,255]
```

Here's what the content of this file should look like.
```ruby
resource_group = attribute('resource_group', default: nil)
￼vault_name     = attribute('key_vault_name', default: nil)
￼
control 'azurerm_key_vaults' do￼
  describe azurerm_key_vaults(resource_group: resource_group) do
    it           { should exist }
    its('names') { should include vault_name }
  end
end
```

Reply:
Yep - an oversight!